app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -20,9 +20,16 @@ def schedule_job_button(*args)
       link_to(_('Schedule Remote Job'), new_job_invocation_path(:host_ids => [args.first.id]), :id => :run_button, :class => 'btn btn-default')
     end
 
+    def web_console_button(href, *args)
+      link_to(_('Web Console'), href + "/=" + args.first.name, :class => 'btn btn-default')
```

Comment:
It would be better if href is a template such as "/webcon/=${host}", instead of just a prefix.

Reply:
Although this might complicate the check in cockpit_redirect.

Reply:
This is done.